source/operations/install-deploy-manage/deploy-minio-tenant.rst

```diff
@@ -565,12 +538,276 @@ the Kubernetes cluster.
+  
+  By default, the Operator uses the release image available at the time of the Operator's release.
+  To specify a different MinIO version for the tenant, such as the latest available, use the :mc:`~kubectl minio tenant create --image` flag.
+
+  See the `MinIO Quay <https://quay.io/repository/minio/minio>`__ or the `MinIO DockerHub <https://hub.docker.com/r/minio/minio/tags>`__ repositories for a list of valid tags.
+
+- :mc:`~kubectl minio tenant create --image-pull-secret`
+           
+  If using a custom container registry, specify the secret to use when pulling the ``minio`` image.
+
+  Use :mc:`~kubectl minio tenant create --image-pull-secret` to specify the secret.
+
+- :mc:`~kubectl minio tenant create --kes-config`
+           
+  Configure a :minio-git:`Key Encrption Service (KES) <kes>`
+
+  Use the :mc:`~kubectl minio tenant create --kes-config` flag to specify the name of the secret to use for KES Key Management Service (KMS) setup.
+
+  Enabling Server Side Encryption (SSE) also deploys a MinIO :minio-git:`KES <kes>` service in the Tenant to faciliate SSE operations.
+  
+  For more, see the `Github documentation <https://github.com/minio/kes/wiki>`__.
+
+
+
+.. note:: Generate a YAML File for Further Customizations
+
+   The MinIO Operator installs a `Custom Resource Definition (CRD) <https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/>`__ to describe tenants.
+   Advanced users can generate a YAML file from the command line and customize the tenant based on the CRD.
+
+   Do a dry run of a tenant creation process to generate a YAML file using the :mc:`~kubectl minio tenant create --output` flag.
+
+   When using this flag, the operator does **not** create the tenant.
+   Modify the generated YAML file as desired, then use ``kubectl apply -f <FILE>`` to manually create the MinIO tenant using the file.
+
+.. _create-tenant-cli-enter-command:
+
+3) Run the Command with Required and Optional Settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+At the command line, enter the full command with all *Required* and any *Optional* flags.
+
+Consider a tenant we want to create:
+
+Tenant Name
+  ``minio1``
+
+Capacity
+  16 Tebibytes
+
+Servers
+  4
+
+Volumes
+  16 (four per node)
+
+Namespace
+  ``miniotenantspace``
+
+MinIO Image
+  Latest version, |minio-latest|
+
+Key ecnryption file
+  ``minio-secret``
+
+Storage class
+  ``warm``
+
+.. code-block:: shell
+   :substitutions:
+
+   kubectl minio tenant create                                \
+                        minio1                                \
+                        --capacity 16Ti                       \
+                        --servers 4                           \
+                        --volumes 16                          \
+                        --namespace miniotenantspace          \
+                        --image |minio-latest|  \
+                        --kes-config minio-secret             \
```

Comment:
```suggestion
                        --kes-config minio-kes-config             \
```

`minio-secret` might be a default secret name for tenant credentials

Reply:
Updated the example